diff --git a/lib/winston/container.js b/lib/winston/container.js
index 683eee3c5160ef66a82e6102f24a5cb973f29eb2..e48b86fb8f3d59746e39a5c2ec9e5e0cb15ddae1 100644
--- a/lib/winston/container.js
+++ b/lib/winston/container.js
@@ -8,7 +8,7 @@
 
 var common = require('./common'),
     winston = require('../winston'),
-    extend = require('util')._extend;
+    extend = Object.assign;
 
 //
 // ### function Container (options)
